readme.md

```diff
@@ -83,6 +84,10 @@ A few exceptions exist to [JSON.stringify][] while using [`toJSON`][] or
   will receive the string `[Circular]` as the argument instead of the circular
   object itself.
 
+A side effect free variation without these limitations can be found as well
```

Comment:
Change to:

For an experimental alternative stable stringify which is faster, side-effect free and without the aforementioned limitations, see  [`safe-stable-stringify`][].

Reply:
one of the reasons we moved it out is because its untested (in terms of adoption) – which I've equated to experimental - I'm all up for an alternative word which conveys the same message as to why there's competing implementations

Reply:
I added another sentence to highlight. PTAL

Reply:
cool - just language tidy up: 

It is still considered experimental due to a new and more complex implementation.